Makefile

```diff
@@ -33,7 +40,6 @@ BUILD_INFO ?= $(shell date +%s)
 LDFLAGS_PODMAN ?= $(LDFLAGS) -X main.gitCommit=$(GIT_COMMIT) -X main.buildInfo=$(BUILD_INFO)
 ISODATE ?= $(shell date --iso-8601)
 LIBSECCOMP_COMMIT := release-2.3
-
```

Comment:
Are you intentionally removing this blank line?  I think the blank line helps to more clearly associate the following comment with its block of Make code.

Reply:
fixed